java/ql/src/Security/CWE/CWE-036/OpenStream.ql

```diff
@@ -0,0 +1,50 @@
+import java
```

Comment:
This needs a documentation header in order for it to be rendered correctly in the query report.

Reply:
Hi @JLLeitschuh! Thanks for the hint. I added a basic documentation header.